src/Devices/LiquidCrystal_TC.cpp

```diff
@@ -86,5 +88,19 @@ void LiquidCrystal_TC::writeLine(const char* text, int line) {
   setCursor(0, line);
   print("                ");
```

Comment:
When I wrote up the issue I didn't realize that we already have code that clears the entire line (my bad!). It seems to me that either the line above isn't needed or the code below isn't needed. Thoughts?

Reply:
I think the code below should stay because it has the capacity to trim larger strings down to 16 characters

Reply:
agreed